SparseGrids/tsgRuleWavelet.cpp

```diff
@@ -332,34 +350,41 @@ inline double RuleWavelet::eval_cubic(int point, double x) const{
             // i.e. 7 or 8
             // These wavelets are reflections across the y-axis of 6 & 5, respectively
             x = -x;
+            sgn = -1.0;
             point = 13 - point;
         }
         point -= 5;
-        return interpolate(&data[2][point*num_data_points],x);
+        return (mode == 0 ? 1.0 : sgn) * interpolate<mode>(&data[2][point*num_data_points],x);
     }else if(l == 3){
         if (point > 12){
             // i.e. 13, 14, 15, 16
             // These wavelets are reflections of 12, 11, 10, 9, respectively
             x = -x;
+            sgn = -1.0;
             point = 25 - point;
         }
         point -= 9;
-        return interpolate(&data[3][point*num_data_points],x);
+        return (mode == 0 ? 1.0 : sgn) * interpolate<mode>(&data[3][point*num_data_points],x);
     }
     // Standard lifted wavelets.
     int subindex = (point - 1) % (1 << l);
     double scale = pow(2,l-4);
-    // Left Boundary
```

Comment:
Is this no longer related to the `Left Boundary`?

Reply:
Each boundary type is now in its own if/else if/else block, so I moved this comment downwards.

Reply:
OK!